### ISSUE TITLE
Use correct dash character

### ISSUE THREAD

**Issue by rgaiacs** [swcarpentry/lesson-example#180]
In https://github.com/swcarpentry/python-novice-inflammation/pull/478, @cleary3 asked to use the correct dash. This is something that doesn't bother me too much but I know it does for others. If we decide to use the correct dash, we **must** do it in all the episodes for all the lessons which means that it should be included in our validation rules.

**Comment by fmichonneau** [comment]
Unless there is an automated way to convert directly dashes into en-dash or em-dash depending on the context, I'm -1 to enforce this strictly across our lessons at this stage.

**Comment by rgaiacs** [comment]
> Unless there is an automated way to convert directly dashes into en-dash or em-dash depending on the context

I don't think that kramdown can do it. And we can't use extensions with GitHub Pages. One of the reasons that I want to use GitLab Pages.

> enforce this strictly across our lessons at this stage.

Before we review the validation rules nothing is going to change. But this is something that we should have in our style guide. Would be strange to read something like

> Programs like the Firefox browser are full of assertions: 10-20% of the code they contain are there to check that the other 80–90% are working correctly.

where the first is a figure dash, `-` or the key that you find on your keyboard, and the second is a en dash, `–` or or Unicode 0x2013 or `&ndash;`. This would decrease our learner experience because would look like that the lesson doesn't have a quality control to apply the same rule to all the text.

**Comment by cleary3** [comment]
Certainly, it's less important to have all the correct dash characters in place than it is to have lessons that don't distract learners because of varying style across the sections of the lesson.

So, if a gradual conversion to the correct dash characters over an admittedly long period of time is going to cause distraction, then you should probably deny my pull request.

Thanks for the thoughtful feedback!
